### DISCUSSION TITLE
Failed to extract pod utilization metrics. Make sure you have a metrics-server deployed on your cluster.

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [nebari-dev/nebari#1250] [Q&A]
A colleague recommended trying out [kdash](https://github.com/kdash-rs/kdash) for monitoring the kube cluster instead of [k9s](https://k9scli.io/).   

When I tried it out, it worked, but spit out this warning/info message:
```
│Failed to extract pod utilization metrics. Make sure you have a metrics-server deployed on your cluster.      │
```
I have no idea what that means, just passing along the info in case it's useful/interesting.

**Comment by costrouc** [comment]
KDash looks nice and thanks for sharing that! Maybe your issue is releated to https://github.com/kubernetes-sigs/metrics-server/issues/963? Not sure. We haven't tweak Kubernetes in any way at the moment that should affect that. My guess is that this would be something aws specific.

**Reply by costrouc** [reply]
Also worth checking out is https://k8slens.dev/

**Reply by costrouc** [reply]
Think it all comes down to if you prefer terminals/guis more.
